src/components/va-radio/test/va-radio.e2e.ts

```diff
@@ -0,0 +1,145 @@
+
+    expect(await options[0].getProperty('checked')).toBeTruthy();
+    expect(await options[1].getProperty('checked')).toBeFalsy();
+
+    await options[1].click();
+
+    expect(await options[0].getProperty('checked')).toBeFalsy();
+    expect(await options[1].getProperty('checked')).toBeTruthy();
+  });
+
+  it('renders an error message if passed', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-radio error="This is an error"></va-radio>');
+
+    const element = await page.find('va-radio >>> .error-message');
+    expect(element).toEqualHtml(`
+     <span class="error-message" role="alert">
+       <span class="sr-only">
+         Error
+       </span>
+       This is an error
+     </span>
+    `);
+  });
+
+  it('renders a required span based on prop', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-radio required></va-radio>');
+
+    const element = await page.find('va-radio >>> .required-span');
+    expect(element).toEqualHtml(`
+      <span class="required-span">
+        (*Required)
+      </span>
+    `);
+  });
+
+  it('fires an analytics event when enableAnalytics is true', async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+      <va-radio label="Regular label" enable-analytics>
+        <va-radio-option label="First option" value="one"></va-radio-option>
+      </va-radio>
+      `);
+    const analyticsSpy = await page.spyOnEvent('component-library-analytics');
+    const inputEl = await page.find('va-radio-option >>> input');
+    await inputEl.click();
+
+    expect(analyticsSpy).toHaveReceivedEventDetail({
+      action: 'change',
+      componentName: 'va-radio',
+      details: {
+        label: 'Regular label',
+        optionLabel: 'First option',
+        required: false,
+      },
+    });
+  });
+
+  it("doesn't fire an analytics event when enableAnalytics is false", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+      <va-radio label="Regular label">
+        <va-radio-option label="First option" value="one"></va-radio-option>
+      </va-radio>
+      `);
+
+    const analyticsSpy = await page.spyOnEvent('component-library-analytics');
+    const inputEl = await page.find('va-radio-option >>> input');
+    await inputEl.click();
+
+    expect(analyticsSpy).not.toHaveReceivedEvent();
+  });
+
+  it('fires a vaValueChange event when the checked value changes', async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+      <va-radio label="Regular label">
+        <va-radio-option label="First option" value="one"></va-radio-option>
+      </va-radio>
+      `);
+
+    const changeSpy = await page.spyOnEvent('vaValueChange');
+    const inputEl = await page.find('va-radio-option >>> input');
+    await inputEl.click();
+
+    expect(changeSpy).toHaveReceivedEvent();
+  });
+
+  it('fires a vaKeyDown event when a key is pressed on focused button', async () => {
```

Comment:
This is maybe more for my edification, but why does this need to happen? What will the `vaKeyDown` event be used for?

Reply:
I'm not really sure... I was just modeling this component after what we already have:

https://github.com/department-of-veterans-affairs/component-library/blob/d71c50001f14751acd5f2b4f1bda4301ef90f535/src/components/RadioButtons/RadioButtons.jsx#L226-L229

Reply:
Putting this here for posterity after slacking.

---

Did some more digging. It looks like at the moment we only have one usage of `<RadioButtons>` with `onKeyDown`:

![image](https://user-images.githubusercontent.com/2008881/128427700-f6fec968-3230-4c0c-a671-14bc97bbbc63.png)

That usage looks like this:

https://github.com/department-of-veterans-affairs/vets-website/blob/b87df585fae073fa2ac4b7d3eb504cf15ea16339/src/applications/coronavirus-vaccination-expansion/containers/IntroductionPage.jsx#L83-L86

Which appears to be a copypaste from storybook:

![image](https://user-images.githubusercontent.com/2008881/128429204-2da492fe-9d71-4735-839e-c0d94d4ec8c6.png)


So, I think it's safe to delete the `vaKeyDown` event.